src/sentry/static/sentry/app/views/settings/account/accountDetails.jsx

```diff
@@ -1,67 +0,0 @@
-import React from 'react';
-
-import {updateUser} from 'app/actionCreators/account';
-import AsyncView from 'app/views/asyncView';
-import AvatarChooser from 'app/components/avatarChooser';
-import Form from 'app/views/settings/components/forms/form';
-import JsonForm from 'app/views/settings/components/forms/jsonForm';
-import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
-import accountDetailsFields from 'app/data/forms/accountDetails';
-import accountPreferencesFields from 'app/data/forms/accountPreferences';
-
-const ENDPOINT = '/users/me/';
-
-class AccountDetails extends AsyncView {
-  getEndpoints() {
-    return [['user', ENDPOINT]];
-  }
-
-  handleSubmitSuccess = user => {
```

Comment:
this was duplicated ... because we were already saving the forms via request form